tests/project/project_1/project.yml

```diff
@@ -0,0 +1,22 @@
+native_app:
+  name: myapp
+
+  # the following shows how to properly use a quoted identifier in YAML
+  source_stage: >-
+    "MySourceSchema"."SRC_Stage"
+
+  artifacts:
+    - dest: ext_code/
+      src:
+        - java/build/*.jar
+        - java/deps/myDep.jar
+    - dest: ext_code/
+      src: python/build/my-package-0.0.0.wheel
+    - setup.sql
+    - manifest.yml
+    - app/README.md
+    - src: app/streamlit/*.py
+      dest: streamlit/
+
+  scripts:
```

Comment:
I'm not sure this is the best name. If we consider allowing "custom scripts" as in `npm run` then scripts key would the best one to use.
```yml
scripts:
   name: build
   cmd: gradle build
```

Reply:
I think these would end up in the top-level of the file, not under the `native_app:` category. But we can definitely come up with a better name. These are SQL scripts that are executed after the app package is created, but before it is installed, to define shared content (i.e. schemas on the package that use reference_usage). Any suggestions?

Reply:
I ended up putting it under "package"; made some sense once we merged the schemas as we already had the key. But I'm also open to `package_scripts`, `package.exec_scripts`.

Could also do `package.shared`?